p2p-specs/p2p-interface.md

```diff
@@ -115,7 +123,7 @@ Bundlers MUST locally store the following `MetaData`:
 ```
 (
   seq_number: uint64
-  supported_mempools: List[Bytes32, MAX_SUPPORTED_MEMPOOLS]
+  supported_mempools: List<mempool_id, MAX_SUPPORTED_MEMPOOLS>
```

Comment:
```suggestion
  supported_mempools: List[mempool_id, MAX_SUPPORTED_MEMPOOLS]
```

Looks like this was copied from my comment, I should have used `[]` instead of `<>`.